doc/man/goto-instrument.1

```diff
@@ -1015,7 +1015,9 @@ force aggressive slicer to preserve all direct paths
 .SS "Code contracts:"
 .TP
 \fB\-\-apply\-loop\-contracts\fR
-check and use loop contracts when provided
+.TP
+\fB\-disable\-side\-effect\-check\fR
```

Comment:
Suggestion `--disable-loop-contract-side-effect-check` makes it more obvious that it is for loop contracts only, not function contracts.

Reply:
Elsewhere, the context may be even less obvious, so +100 to what Remi said.